runtime/su.ts

```diff
@@ -72,7 +72,7 @@ export function get(x: any, key: any): any {
     if (x instanceof SuValue) {
         let result = x.get(key);
         if (result == null)
-            throw new Error("uninitialized member: " + display(key));
+            throw new Error("uninitialized member not found: " + display(key));
```

Comment:
I think we could go directly to just "member not found"
since all the library code has now been updated.

Reply:
I have changed the message in the new commit